### DISCUSSION TITLE
[README] Use special blockquotes from GitHub

### DISCUSSION THREAD

**Discussion by Andre601** [simple-icons/simple-icons#7429] [General]
I recently discovered that GitHub added a feature where using `**Note**` or `**Warning**` as the first line of a Block quote results in that line being rendered differently, allowing more "informative" block quotes.

Maybe this repository could benefit from this small visual change?
I've gone ahead and updated the block quotes in the readme on my fork, so that you can see how it could look like.

Link: https://github.com/Andre601/simple-icons/tree/feature/use-github-special-quotes

I made the note and warning line separate, but from my testing could it also be a single line, where the Note/Warning would be set as a sort of prefix.

Example:
![image](https://user-images.githubusercontent.com/11576465/169415035-7e480426-e5e2-4e5c-8a98-e6b9b0901fdc.png)
![image](https://user-images.githubusercontent.com/11576465/169415051-2ae63f1e-f8de-4427-bf8a-db44fb80c26e.png)

**Comment by LitoMore** [comment]
@Andre601 Let's use a single line, just like what I did at https://github.com/microsoft/terminal/pull/13615. PRs welcome.

**Comment by adamrusted** [comment]
Addressed in #8974. Closing this discussion.
